cypress/e2e/registration.spec.js

```diff
@@ -0,0 +1,24 @@
+import { buildUser } from "../support/generate";
```

Comment:
what about calling this file signup instead of registration

Reply:
We can do so.